src/main/java/seedu/club/logic/commands/EditCommand.java

```diff
@@ -40,6 +40,8 @@
 public class EditCommand extends UndoableCommand {
 
     public static final String COMMAND_WORD = "edit";
+    public static final String COMMAND_FORMAT = "edit [n/NAME] [p/PHONE] [e/EMAIL] [m/MATRIC NUMBER]"
```

Comment:
Just my 2 cents, the autocomplete is supposed to increase our speed of typing commands. If autocomplete fills the commandbox with [n/NAME] [p/PHONE] [e/EMAIL], it would slow users down because we have to delete the template and type in the real values. Maybe just autocompleting the command word itself would be more intuitive, e.g "ed" autocompletes to "edit" instead of "edit [n/NAME] [p/PHONE]......"

Reply:
how about i autocomplete the thing to `edit [n/ ] [p/ ] ....` ? Basically, i'll allow the user to fill in the blanks.
My idea of auto-completing the command this way is that the user will be shown the valid format of the command to minimize invalid command formats

Reply:
sure seems good